scenes/game_elements/props/collectible_item/components/collectible_item.gd

```diff
@@ -83,8 +86,10 @@ func reveal() -> void:
 func _on_interacted(player: Player, _from_right: bool) -> void:
 	z_index += 1
 	animation_player.play("collected")
+	collectible_sound.stream = collect_sound
+	collectible_sound.play()
 	await animation_player.animation_finished
-
+	await collectible_sound.finished
 	GameState.add_collected_item(item)
```

Comment:
I don't think we should wait for the sound to finish before adding the item to the HUD. It looks weird to me – the item disappears, there's a pause, and then the HUD updates.